proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -39,6 +39,13 @@ message TxResponse {
   // the timestamps of the valid votes in the block.LastCommit. For height == 1,
   // it's genesis time.
   string timestamp = 12;
+  // Events defines all the events emitted by processing a transaction. Note,
+  // these events include those emitted by processing all the messages and those
+  // emitted from the ante handler. Whereas Logs contains the events, with
+  // additional metadata, emitted only by processing the messages.
+  //
+  // Since: cosmos-sdk 0.45
+  repeated tendermint.abci.Event events = 13 [(gogoproto.nullable) = false];
 }
```

Comment:
I'd like to hear @webmaster128's opinion about backporting a new proto field (also pinged him on Discord)

If yes, we can remove the `Since:` line.

Reply:
All good, as long as we're adding the correct `Since:` version

Reply:
Thanks for considering backporting. This will make ensure a consistent experience for multi-chain clients that serve different backend versions at the same time.